sdk/lib/drivers/libusb/libusb.cpp

```diff
@@ -18,6 +18,25 @@
 //
 DRIVER_ADD_DEVICE USBLIB_AddDevice;
 
+PVOID
```

Comment:
Why is this needed, while it wasn't before?

Reply:
tl;dr: read the commit message.

Reply:
The commit message is `[REACTOS] Fix many 64 bit warnings `...

Reply:
Oops. The reson is that newer MSVC versions require (the non-member) operator new and delete to be in the global namespace and not be static or inline. See https://msdn.microsoft.com/en-us/library/mt723604.aspx ("Overloaded operator new and operator delete") I will separate the commit and add a proper commit message.